packages/plugin-html-swipe-response/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@jspsych-contrib/plugin-html-swipe-response",
+  "private": "true",
```

Comment:
I think the private flag will cause an issue with the release.

Reply:
Thanks! Resolved in [aae94fa](https://github.com/jspsych/jspsych-contrib/pull/36/commits/aae94fa279443a90d2768c8b33e3c5c69c781ec1)